doc/user_guide/transform/aggregate.rst

```diff
@@ -70,3 +159,39 @@ class, which has the following options:
 The :class:`~AggregatedFieldDef` objects have the following options:
 
 .. altair-object-table:: altair.AggregatedFieldDef
+
+.. _agg-func-table:
+
+List of Aggregation Functions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In addition to ``count`` and ``average``, there are a large number of available
+aggregation functions built into Altair; they are listed in the following table:
+
+=========  ===========================================================================  =====================================
+Aggregate  Description                                                                  Example
+=========  ===========================================================================  =====================================
+argmin     An input data object containing the minimum field value.                     N/A
+argmax     An input data object containing the maximum field value.                     :ref:`gallery_line_chart_with_custom_legend`
+average    The mean (average) field value. Identical to mean.                           :ref:`gallery_layer_line_color_rule`
+count      The total count of data objects in the group.                                :ref:`gallery_simple_heatmap`
```

Comment:
Vega-Lite [docs](https://vega.github.io/vega-lite/docs/aggregate.html#ops) also state
> Note: ‘count’ operates directly on the input objects and return the same value regardless of the provided field.

Just mentioning in case it's worth adding here as well?

Reply:
> Vega-Lite [docs](https://vega.github.io/vega-lite/docs/aggregate.html#ops) also state
> 
> > Note: ‘count’ operates directly on the input objects and return the same value regardless of the provided field.
> 
> Just mentioning in case it's worth adding here as well?

Maybe that phrasing could replace 
> "... in the other axis" (https://github.com/vega/altair/pull/2655#discussion_r1896348194)